packages/reaction-product-variant/client/templates/products/productGrid/content/content.js

```diff
@@ -4,6 +4,8 @@
 
 Template.gridContent.helpers({
   displayPrice: function () {
-    return this._id && this.price.range;
+    if (this.price && this.price.range) {
```

Comment:
I suppose this.price is always object and it will always coerced to true value. Or there are cases, when it's equal to null or undefined ?